aws-sqs-http-connector/workflow/kubernetes/Dockerfile

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
We are using localstack for testing sqs connector and to connect to localstack from our deployment we require awscli, since our sqs connector images does not contain awscli so i have created a separate image with all the required things to connect to localstack.

There can be different ways of doing this, i followed this approach